client/src/components/Input.js

```diff
@@ -0,0 +1,19 @@
+import styled from '@emotion/styled';
+
+const Input = styled.input`
+  display: flex;
+  justify-content: center;
+  padding: 2px 8px;
+  color: #bc325b;
+  height: 25px;
```

Comment:
You work here with px for the height. Leon has comment this in a pull req. from Flo. We should work with flex basis, shrink etc. I am Searing for the comment and will send it to you

Reply:
https://github.com/florianGierlichs/mycrypt/pull/24